src/manifests_workflow/input_manifests_opensearch.py

```diff
@@ -0,0 +1,99 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import logging
+import os
+import subprocess
+
+from manifests.input_manifest import InputManifest
+from manifests_workflow.component_opensearch import ComponentOpenSearch
+from manifests_workflow.component_opensearch_min import ComponentOpenSearchMin
+from manifests_workflow.input_manifests import InputManifests
+from system.temporary_directory import TemporaryDirectory
+
+
+class InputManifestsOpenSearch(InputManifests):
```

Comment:
Why 2 different classes?  seems like this could be done with 2 classes that inherit or compose from a standard set of instructions